src/libImaging/Unpack.c

```diff
@@ -480,86 +479,78 @@ void
 ImagingUnpackRGB(UINT8* _out, const UINT8* in, int pixels)
 {
     int i = 0;
-#ifdef __sparc
-    /* SPARC CPUs cannot read integers from nonaligned addresses. */
```

Comment:
Are the removals of these exceptions for SPARC intentional?

They were only added recently, in https://github.com/python-pillow/Pillow/pull/3858.

Reply:
Yes, by using memcpy the compiler will care for the correct access pattern.

Reply:
OK, thanks.

cc @kulikjak fyi.